docs/api/tutorials/Player Tips and Tricks.md

```diff
@@ -0,0 +1,35 @@
+The **Player Tips and Tricks** section serves as a valuable resource for users seeking guidance on optimizing their experience with Pillarbox.
+
+## Adding or Overriding Resource Properties
+
+There are scenarios where developers need to customize resource properties or override existing ones. This section provides insights into achieving both objectives effectively.
+
+### Example
+
+Consider the following use case:
+
+1. **Adding a Custom Property**:
+   - Sometimes, you may want to introduce a custom property to a resource. This could involve adding metadata, specifying additional behavior, or enhancing the resource's functionality.
+
+2. **Overriding an Existing Property**:
+   - In other cases, you might need to modify an existing property. For instance, you could override the media source.
```

Comment:
Is it possible to delete a property ? e.g. if I explicitly make a property `null` or `undefined`, what is the expected result ?

Reply:
It will be replaced by any value passed to `mediaData`, since it uses `videojs.obj.merge`, which is the expected behavior.